src/PendingCalls/TestCall.php

```diff
@@ -168,6 +172,63 @@ public function skip(Closure|bool|string $conditionOrMessage = true, string $mes
         return $this;
     }
 
+    /**
+     * Sets the covered classes or methods.
+     */
+    public function covers(string ...$classesOrFunctions): TestCall
+    {
+        foreach ($classesOrFunctions as $classOrFunction) {
+            $isClass  = class_exists($classOrFunction);
+            $isMethod = function_exists($classOrFunction);
+
+            if (!$isClass && !$isMethod) {
+                throw new InvalidArgumentException(sprintf('No class or method named "%s" has been found.', $classOrFunction));
+            }
+
+            if ($isClass) {
```

Comment:
Nice! Definitely like having the single `covers()` method as an option. 👍🏻

Wonder if it's worth doing some early `continue` uses in here, not sure though.

Reply:
Got your point, you tell me what would you prefer to see, at the end it's the same 😄

Reply:
I think it's good to leave as is. 👍🏻